packages/builder/src/builders/webpack.js

```diff
@@ -0,0 +1,297 @@
+import path from 'path'
```

Comment:
@clarkdo Should we keep this file into `builder` package or still a part of WIP?

Reply:
It’s not finished yet 😊

Reply:
Forgive me. I'm too excited to see the progress 💃